publiccode.yml

```
@@ -0,0 +1,89 @@
+# This repository adheres to the publiccode.yml standard by including this
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
+name: 'itowns'
+url: 'https://github.com/iTowns/itowns'
+landingURL: 'https://github.com/iTowns/itowns.git'
```

According to the specification, the `url` key
> [...] must be a URL to the source code repository (git, svn, …)

and `landingURL` a
> `[...] human-readable or browsable page. [...] This page, ideally, is where your users will land when they will click a button labeled something like “Go to the application source code”.

```suggestion
url: 'https://github.com/iTowns/itowns.git'
landingURL: 'https://github.com/iTowns/itowns'
```